content/OTP/Guides/OTP_Walk-through.adoc

```diff
@@ -241,16 +245,24 @@ If the YubiKey ID and User ID do not match, send an error message. Create an app
 
 ==== Inspecting the Code
 
+
 Verify your client is compatible with your application code base and language.
 
 
+
 === Wrapping Up
+
 Congratulations! You've completed all the steps to enable your users to register and authenticate with an OTP credential.
 
 === Additional Resources
 
+==== Help, I'm Stuck!
+
+If you get stuck, you can check link:https://stackoverflow.com[Stack Overflow]. If you don't receive an answer, or remain stuck, please file an issue or open a support ticket and we'll help you out.
+
 ==== Plug-ins for Creating your YubiKey OTP Module
 
+
 View and download the relevant plug-in components.
 
 * link:https://developers.yubico.com/yubico-pam/[Yubico PAM module] – Pluggable Authentication Modules (PAM) for GNU/Linux, Solaris and Mac OS X for user authentication. -- Requires
```

Comment:
Chris Halos: Mac OS X = fwiw we've replaced this with macOS pretty much everywhere since it's no longer referred to as OS X as of 10.11 I believe